tests/scripts/fetch-csi-operator-files.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+TARGET_DIR="deploy/examples/csi-operator"
+DIRECTORIES=("config/crd/bases" "config/rbac")
```

Comment:
What if we were to combine the files to reduce the number of manifests in the rook repo? 
- `config/crd/bases/*` --> `crds.yaml`
- `config/rbac/*` --> `rbac.yaml`

The advantage is that later we anyway want to merge the manifests into the rook manifests of these names. 

But we could also delay this work until later if desired, since it's easy enough for someone to create all the manifests in the folder with a single command.

Reply:
agreed on this, updated